lib/project_types/rails/commands/serve.rb

```diff
@@ -22,6 +22,9 @@ def call(*)
         )
         if @ctx.mac? && project.env.shop
           @ctx.puts("{{*}} Press {{yellow: Control-T}} to open this project in {{green:#{project.env.shop}}} ")
+
+          # Reset any previous SIGINFO handling we had so the only action we take is opening the URL
+          trap('INFO', 'DEFAULT')
```

Comment:
Should this be done in the context method rather than repeated across commands?

Reply:
It should. I checked and this method isn't used anywhere else, so the repetition is pointless. Fixed in 0fa73e2.